crates/re_space_view_spatial/src/ui_3d.rs

```diff
@@ -92,10 +92,13 @@ fn ease_out(t: f32) -> f32 {
 impl View3DState {
     pub fn reset_camera(
         &mut self,
-        scene_bbox_accum: &BoundingBox,
+        scene_bbox: &SceneBoundingBoxes,
         scene_view_coordinates: Option<ViewCoordinates>,
     ) {
-        self.interpolate_to_orbit_eye(default_eye(scene_bbox_accum, scene_view_coordinates));
+        // Mark as interaction since we want to stop doing any automatic interpolations,
+        // even if this is caused by a full reset.
+        self.last_eye_interaction = Some(Instant::now());
```

Comment:
When tracking an entity (camera), `reset_camera` is called each frame, I believe.

Won't resetting `last_eye_interaction` each frame cause the center of the orbit camera to show up each frame?

Also, what does this have to do with the PR description?

Reply:
Originally I also wanted that thing
```
        if self.last_eye_interaction.is_none() {
            self.interpolate_to_orbit_eye(default_eye(
                &bounding_boxes.accumulated,
                scene_view_coordinates,
            ));
        }
```
to use the current bounding box, but in many example scenes this looks too jerky